src/components/widgets/Bolk/Bolk.jsx

```diff
@@ -1,21 +1,28 @@
 import React from 'react';
 import Uenigboks from '../Uenigboks';
 import IkonHeader from '../IkonHeader';
+import PropTypes from 'prop-types';
 import './Bolk.css';
 
-const Bolk = ({ title, items }) => {
+const Bolk = ({ title, titleValue, items, ikon = true }) => {
```

Comment:
Hvis ikon skal være default true for `IkonHeader` er det kanskje bedre å sette dette som en del av defaultProps for komponenten, som her:

```javascript
IkonHeader.defaultProps = {
    ikon: true
}
```

Reply:
Da trenger den ikke lenger være required, så `.isRequired` kan fjernes fra `IkonHeader.propTypes`.